modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/shared/ClayIcon.es.js

```diff
@@ -26,6 +26,7 @@ class ClayIcon extends Component {
 			<svg
 				aria-hidden="true"
 				className={classes}
+				key={iconName}
```

Comment:
Why is there a `key` here? `key` should only be required for a list of items.

Reply:
In the Portal, there's a tool that reads the `<use xlinkHref="...` tag and it replaces it with the content referenced in the `xlinkHref` tag. That makes React loose track of the element. So that tag says to React: when the iconName changes, render a new `svg` element, don't update it.

Reply:
Hmm, I see. That is a bit strange that we have to keep it around for that other tool. I guess we can keep it around, but can you also add some code comment to it so that future people know why we use key? Just so we can avoid accidental bugs in case someone reading it in the future removes it by accident.